Apps/Immunization/test/unit/Services.Test/VaccineStatusServiceTests.cs

```diff
@@ -99,15 +110,295 @@ public void ShouldGetVaccineStatus()
+        {
+            RequestResult<PHSAResult<VaccineStatusResult>> delegateResult = new()
+            {
+                ResultStatus = Common.Constants.ResultType.ActionRequired,
+                ResourcePayload = new PHSAResult<VaccineStatusResult>()
+                {
+                    LoadState = new PHSALoadState() { RefreshInProgress = false, BackOffMilliseconds = 500 },
+                    Result = new VaccineStatusResult()
+                    {
+                        FirstName = "Bob",
+                        LastName = "Test",
+                        Birthdate = this.dob,
+                        StatusIndicator = "DataMismatch",
+                    },
+                },
+            };
+            JWTModel jwtModel = new()
+            {
+                AccessToken = this.accessToken,
+            };
+
+            RequestResult<VaccineStatus> expectedResult = new RequestResult<VaccineStatus>()
+            {
+                ResultStatus = delegateResult.ResultStatus,
+                ResourcePayload = new VaccineStatus()
+                {
+                    Loaded = true,
+                    RetryIn = 0,
+                    PersonalHealthNumber = this.phn,
+                    FirstName = "Bob",
+                    LastName = "Test",
+                    Birthdate = this.dob,
+                    State = VaccineState.DataMismatch,
+                },
+                ResultError = new()
+                {
+                    ActionCode = ActionType.DataMismatch,
+                },
+            };
+
+            Mock<IVaccineStatusDelegate> mockDelegate = new Mock<IVaccineStatusDelegate>();
+            mockDelegate.Setup(s => s.GetVaccineStatus(It.IsAny<VaccineStatusQuery>(), this.accessToken, true)).Returns(Task.FromResult(delegateResult));
+
+            Mock<IAuthenticationDelegate> mockAuthDelegate = new Mock<IAuthenticationDelegate>();
+            mockAuthDelegate.Setup(s => s.AuthenticateAsUser(It.IsAny<Uri>(), It.IsAny<ClientCredentialsTokenRequest>())).Returns(jwtModel);
+
+            IVaccineStatusService service = new VaccineStatusService(
+                this.configuration,
+                new Mock<ILogger<VaccineStatusService>>().Object,
+                mockAuthDelegate.Object,
+                mockDelegate.Object,
+                GetMemoryCache(),
+                this.GetHttpContextAccessor().Object);
+
+            string dobString = this.dob.ToString("yyyy-MM-dd", CultureInfo.CurrentCulture);
+            string dovString = this.dov.ToString("yyyy-MM-dd", CultureInfo.CurrentCulture);
+            var actualResult = Task.Run(async () => await service.GetPublicVaccineStatus(this.phn, dobString, dovString).ConfigureAwait(true)).Result;
+            Assert.Equal(Common.Constants.ResultType.ActionRequired, actualResult.ResultStatus);
+            Assert.Equal(expectedResult.ResultError.ActionCode, actual: actualResult.ResultError?.ActionCode);
```

Comment:
I don't think using a named parameter (`actual:`) is required here.  There are 3 places in the file where `actual:` can be removed.